core-bundle/assets/controllers/toggle-fieldset-controller.js

```diff
@@ -59,20 +60,24 @@ export default class extends Controller {
     connect () {
         if (this.element.querySelectorAll('label.error, label.mandatory').length) {
             this.element.classList.remove(this.collapsedClass);
+            this.element.querySelector('button').ariaExpanded = 'false';
```

Comment:
what if the button does not exist?

Reply:
Good catch, we could use something like this as well, wdyt?
```js
this.element.querySelector('button')?.ariaExpanded = 'false';
```

Reply:
@aschempp this is fixed in e882f0405c70dc2a62b4b3a81a38ad77b944d83d
You made me realize there was one case that wasn't added properly.

```js
    setAriaAttributes(state) {
        const button = this.element.querySelector('button');

        if (button) {
            button.ariaExpanded = state;
        }
    }
```